### ISSUE TITLE
Some details about the paper

### ISSUE THREAD

**Issue by lebron-2016** [dongkwonjin/OMR#1]
Dear author,

I have recently studied your work and benefited a lot from it. I would like to ask you for some implementation details.

First, in the implementation of ConvLSTM, do you initialize the hidden state and cell state using nn.Embedding or all-zero initialization?

Secondly, if I understand correctly, you first train the single-frame model and then perform sequence training (learning OMR) during training. What are the implementation details of sequence training? That is, do you read multiple frames of continuous data in one iter to train OMR or read continuous frames between iters for training (in this case, h and c need to be detached())? If it is the first solution, the length of the sequence that can be read is limited due to the limitation of memory. How to ensure the performance of the model under longer sequences during testing? If it is the second solution, since the detach operation is required, can the model learn the representation of h and c well?

Looking forward to your reply.

Thanks!!

**Comment by lebron-2016** [comment]
Hi, I have another key question: when you do inference for each frame, do you re-loop and process its previous 7 frames? Instead of what I understand above, when doing inference/test, you only process the current frame and then pass the hidden state backwards? If this is the case, will it cause the model inference speed to be very slow?
In fact, since `out['prev_num'] = len(self.datalist_video[self.datalist[idx]]['past'])`,`for j in range(len(batch['prev_num'])):` feels more like traversing the batch size (eg.batch['prev_num']=[7,7,7,7], bs=4) rather than a sequence. I hope to get your answer.
![image](https://github.com/user-attachments/assets/bce32d75-872e-4b3a-9a56-97089ecafcae)

Besides, I don't quite understand the meaning of "batch_size = cfg.batch_size['img'] * 4" in the test set.
![image](https://github.com/user-attachments/assets/ca31ea73-ea28-49fc-9147-eba60e77cd0b)

Thanks.

**Comment by lebron-2016** [comment]
Hi, sorry to bother you again. 
Can you explain the meaning of `num_t`? In addition, according to `clip_length=8`, did you read in 8 consecutive frames for processing in one iter during training? Will the setting of `clip_length` have a significant impact on the test results?

![image](https://github.com/user-attachments/assets/d8a18901-18e5-4908-a3a7-b9d1b3f4364d)

Thanks.

**Comment by jhzilongD** [comment]
so many questions，haha
